bw2io/strategies/ecospold2.py

```diff
@@ -236,10 +730,58 @@ def set_lognormal_loc_value(db):
 
 
 def fix_unreasonably_high_lognormal_uncertainties(db, cutoff=2.5, replacement=0.25):
-    """Fix unreasonably high uncertainty values.
-
+    """
```

Comment:
Here the initial short decription of the function is missing.

Reply:
resolved